#!/usr/bin/env python
PACKAGE = "wpb_mani_tutorials"
  
from dynamic_reconfigure.parameter_generator_catkin import *
  
gen = ParameterGenerator()

gen.add( "x_min", double_t,  0,     "x_min",  0.5,      0,       2)
gen.add( "x_max", double_t,  0,     "x_max",  1.5,      0,       2)
gen.add( "y_min", double_t,  0,     "y_min",  -0.5,      -2,       2)
gen.add( "y_max", double_t,  0,     "y_max",  0.5,      -2,       2)
gen.add( "z_min", double_t,  0,     "z_min",  0.1,      0,       2)
gen.add( "z_max", double_t,  0,     "z_max",  0.5,      0,       2)

exit(gen.generate(PACKAGE, "wpb_mani_tutorials", "pc_pass_through")) 